test/net/sourceforge/kolmafia/maximizer/MaximizerTest.java

```diff
@@ -2045,4 +2045,81 @@ public void suggestCardSleeveSlot() {
       }
     }
   }
+
+  @Nested
+  public class Weapons {
```

Comment:
there are already some tests in `Effective` testing this -- the ones where it equips no weapon are new, but the others overlap